pyrovision/models/pyronear_model.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
Let's rename the module to rexnet.py and name the function rexnet1_0x (pyronear_model is not informative)